src/controllers/dashBoardController.spec.js

```diff
@@ -0,0 +1,698 @@
+// const mongoose = require('mongoose');
+const AIPrompt = require('../models/weeklySummaryAIPrompt');
+const { mockReq, mockRes, assertResMock } = require('../test');
+const UserProfile = require('../models/userProfile');
+
+jest.mock('../utilities/emailSender');
+const emailSender  = require('../utilities/emailSender');
+
+jest.mock('../helpers/dashboardhelper');
+const dashboardHelperClosure = require('../helpers/dashboardhelper');
+const dashBoardController = require('./dashBoardController');
+
+// mock the cache function before importing so we can manipulate the implementation
+// jest.mock('../utilities/nodeCache');
+// const cache = require('../utilities/nodeCache');
+const makeSut = () => {
+  const { 
+    updateCopiedPrompt,
+    getPromptCopiedDate, 
+    updateAIPrompt, 
+    getAIPrompt,
+    monthlydata,
+    weeklydata,
+    leaderboarddata,
+    orgData,
+    dashboarddata,
+    sendBugReport,
+    sendMakeSuggestion,
+    // getSuggestionOption
+  } = dashBoardController(AIPrompt);
+  return { 
+    updateCopiedPrompt,
+    getPromptCopiedDate,
+    updateAIPrompt,
+    getAIPrompt,
+    monthlydata,
+    weeklydata,
+    leaderboarddata,
+    orgData,
+    dashboarddata,
+    sendBugReport,
+    sendMakeSuggestion,
+    // getSuggestionOption
+  };
+};
+
+
+const flushPromises = async () => new Promise(setImmediate);
+
+describe('Dashboard Controller tests', () => {
+  beforeAll(() => {
+    // const dashboardHelperObject = 
+    //   {
+    //     laborthisweek: jest.fn(() => Promise.resolve([]))
+    //   };
+    // dashboardHelperClosure.mockImplementation(() => dashboardHelperObject);
+  });
+  beforeEach(() => {
+    dashboardhelper = dashboardHelperClosure();
```

Comment:
Hey Lin,
If the variable `dashboardhelper` isn't used in the code, is it possible to get rid of it?

Reply:
Hey Parth! Thanks for your suggestion.